py-polars/tests/parametric/test_series.py

```diff
@@ -10,24 +10,6 @@
 from polars.testing import assert_series_equal
 from polars.testing.parametric import series
 
-# TODO: once Decimal is a little further along, start actively probing it
```

Comment:
I removed this TODO, because we already have tests for `to_list`, and for the strategy itself. More tests are always welcome, but don't have to be a TODO like this.